src/Components/SingleSighting.js

```diff
@@ -52,18 +76,23 @@ const SingleSighting = () => {
         </div>
       </div>
       <div className='sighting-info-container'>
-      <div className='sighting-details'>
-        <p className='sighting-title'>{data.sightingById.title}</p>
-        <p className='sighting-location'>Location: {data.sightingById.location}</p>
-        <p className='sighting-description'>{data.sightingById.description}</p>
-        <p className='near-text'>Nearby Trails:</p>
+        <div className='sighting-details'>
```

Comment:
This scroll with the conditional arrow looks great!

Reply:
^^^